dotnet/src/Experimental/Agents/Internal/ChatThread.cs

```diff
@@ -50,33 +50,31 @@ public static async Task<IAgentThread> GetAsync(OpenAIRestContext restContext, s
     }
 
     /// <inheritdoc/>
-    public async Task<IChatMessage> AddUserMessageAsync(string message, CancellationToken cancellationToken = default)
+    public async Task<IChatMessage> AddUserMessageAsync(string message, IEnumerable<string>? fileIds = null, CancellationToken cancellationToken = default)
     {
         this.ThrowIfDeleted();
 
-        var messageModel =
-            await this._restContext.CreateUserTextMessageAsync(
-                this.Id,
-                message,
-                cancellationToken).ConfigureAwait(false);
+        var messagemodel = fileIds == null
```

Comment:
Can't it just be simplified to - "var messageModel = await this._restContext.CreateUserTextMessageAsync(this.Id, message, **fileIds,**, cancellationToken).ConfigureAwait(false);" ?

Reply:
Yes, both comments are a side-effect of my edits on top of Ryan's PR.